Entities/Characters/Scripts/RunnerDefault.as

```diff
@@ -26,6 +26,7 @@ void onInit(CBlob@ this)
 
 void onTick(CBlob@ this)
 {
+	this.Untag("crouch dodge");
```

Comment:
Are you untagging and retagging every frame? That doesn't seem great.

Reply:
What should I be doing instead? Should I untag in every animation state that doesn’t allow crouch dodging?

Reply:
probably need to tag / untag only when an animation state that doesn't allow starts / ends

Reply:
That’ll require lots of repetitive lines of code...